lib/blockchainState/BlockchainState.js

```diff
@@ -3,15 +3,18 @@ const Long = require('long');
 class BlockchainState {
   /**
    *
+   * @param {number} [appVersion] - Protocol version
```

Comment:
What is the point to store protocol version in the state? I guess we can just hardcode it in DI container.

Reply:
Would you be fine with using env var instead of hardcoding it in DIContainer ? 
(I would then import them the same way we do for all other envs)

Reply:
I guess nobody should control it. It depends on the code only so I would keep it in code as a constant (like we are doing it in Core),